examples/bento/no-v0.html

```diff
@@ -0,0 +1,391 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>amp-timeago:1.0, no v0.js</title>
+    <link rel="canonical" href="amps.html" />
+    <meta
+      name="viewport"
+      content="width=device-width,minimum-scale=1,initial-scale=1"
+    />
+    <!-- <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript> -->
+    <style amp-boilerplate>
+      [layout='fixed'] {
+        display: inline-block;
+        position: relative;
+      }
+    </style>
+    <style amp-custom>
+      body {
+        font-family: sans-serif;
+        max-width: 400px;
+        font-size: 16px;
+        line-height: 20px;
+      }
+      .scroll {
+        height: 40px;
+        overflow: scroll;
+        border: 1px solid black;
+      }
+    </style>
+    <script src="https://cdn.ampproject.org/custom-elements-polyfill.js"></script>
+    <script
+      async
+      custom-element="amp-timeago"
+      src="https://cdn.ampproject.org/v0/amp-timeago-1.0.js"
+    ></script>
+  </head>
+  <body>
+    <h1>amp-timeago:1.0, no v0.js</h1>
+    <h2>Basic example</h2>
+    <div class="scroll">
+      <p>Scroll up and down for updated times</p>
+      <amp-timeago
+        class="time"
+        layout="fixed"
+        width="160"
+        height="20"
```

Comment:
Can't `layout`, `width` and `height` attrs all be removed?

Reply:
The `CustomElement` implementation will `applyStaticLayout` as always. Why wouldn't we want these attributes?

Reply:
Discussed offline. We will not apply static layout.